### ISSUE TITLE
Google Play doesn't work ("you're offline")

### ISSUE THREAD

**Issue by Bent3ndo** [TheGammaSqueeze/GammaOS#138]
Hi; I've tried using the Google Play store multiple times, and have tried removing all updates etc and deleted cashe etc as well, and no matter what I do the Play store refuses to connect and says "you're offline".

**Comment by dfreivald** [comment]
Any more info? 

- Which version of GammaOS
- Is this a fresh install
- Did you run wipeuserdata

**Comment by Bent3ndo** [comment]
The latest version of GammaOS for the RG405V (has been the case ever since I moved from stock firmware however so hasn't just been the current version).

The beta of GammaOS was a clean install and then I updated through the official way.

And I've not run any erase options or programs at any point since GammaOS has been installed. The most I've done it remove updates to see if reverting back to the original version included helped, but it doesn't.
